### PR TITLE
feat(dashmate): remove sentinel service

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Sentinel is now implemented as part of DashCore so sentinel service is deprecated and needs to be removed

## What was done?

- Remove sentinel
- Add `deprecatedrpc` to dash.conf

## How Has This Been Tested?

Locally with fullnodes/masternodes

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — COMMENTED** [review]
We want to disable sentinel even if the node is a masternode?

**Comment by strophy** [comment]
Yes, that was the request

**Comment by thephez** [comment]
> We want to disable sentinel even if the node is a masternode?

Yes, this corresponds to changes in Core v20 to pull sentinel functionality directly into Core (https://github.com/dashpay/dash/pull/5525)

**Review by pshenmic — DISMISSED** [review]
LGTM, tested locally

**Review by pshenmic — APPROVED** [review]

**Review by ogabrielides — APPROVED** [review]
LGTM

**Review by antouhou — APPROVED** [review]
Looks good to me 👍

File: packages/dashmate/configs/getConfigFileMigrationsFactory.js [hunk 1]
```diff
@@ -124,6 +124,13 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
           });
         return configFile;
       },
+      '0.25.0': (configFile) => {
```
Comment (strophy) [inline packages/dashmate/configs/getConfigFileMigrationsFactory.js:127]: Is this the right way to handle migrations for `-dev` releases?

Comment (shumkov) [reply packages/dashmate/configs/getConfigFileMigrationsFactory.js:127]: You can set 0.25.0-dev.18

File: packages/dashmate/docker-compose.yml [hunk 2]
```diff
@@ -40,25 +40,6 @@ services:
       - core
       - masternode
```
Comment (shumkov) [inline packages/dashmate/docker-compose.yml:41]: You don't need profile masternode anymore. Please check the generateEnvs function and update logic there as well

**Review by shumkov — CHANGES_REQUESTED** [review]

**Pull request merged**